src/firefly/js/tables/ui/TablePanel.jsx

```diff
@@ -313,16 +313,15 @@ function Loading({showTitle, tbl_id, title, removable, bgStatus}) {
 }
 
 function TableError({tbl_id, message}) {
-    const {request} = TblUtil.getTblById(tbl_id);
-    const canReset = get(request, 'filters') || get(request, 'sortInfo');
+    const prevReq = TblUtil.getResultSetRequest(tbl_id);
     const reloadTable = () => {
-        const origRequest = omit(cloneDeep(request), 'filters', 'sortInfo');
-        dispatchTableSearch(origRequest);
+        dispatchTableSearch(JSON.parse(prevReq));
     };
     return (
         <div className='TablePanel__error'>
+            <div><b>Table Load Error:</b></div>
             <div>{message}</div>
-            {canReset && <button type='button' className='button std' onClick={reloadTable}>Reload</button>}
+            {prevReq && <button type='button' className='button std' onClick={reloadTable}>Back</button>}
```

Comment:
What is the 'back' button doing? Don't you mean 'OK' or some user reading confirmation?

Reply:
This error usually shows up when you've entered a bad filter or sort condition.  The 'Back' button is to put the table back to the state prior to this bad operation.  If you have a better word, let me know.  I have no problem changing it.